src/aura/RD2_EntryForm/RD2_EntryFormHelper.js

```diff
@@ -0,0 +1,112 @@
+({
+    /**
+     * @description: Create the modal when initializing the component
+     */
+    handleShowModal: function (component, event, helper) {
+        let recordId = component.get('v.recordId');
+        let parentId;
+
+        if (!recordId) {
+            parentId = helper.getParentId();
+            component.set('v.parentId', parentId);
+        }
+        
+        $A.createComponent("c:rd2EntryForm", {parentId, recordId},
+        function(content, status, errorMessage) {
```

Comment:
I was experimenting with creating a new schedule LWC, and the modal was failing, and the error message was not logged. So, let's call the `errorMessage` as `error` and replace lines 28-30 with:
```
} else {
    console.error(JSON.stringify(error));
}
```